include/common/aarch32/el3_common_macros.S

```diff
@@ -195,6 +195,18 @@
 	cps	#MODE32_mon
 	isb
 
+#if defined(IMAGE_BL32) && SP_MIN_WITH_SECURE_FIQ
```

Comment:
ok. i see your idea. 
If you feel this implementation better suites the atf generic way, I am fine.

A drawback a see is to clearly define what such `_route_fiq` argument would mean on AArch64 version of the macro `el3_entrypoint_common`. And digressions on fiq/irq/where-are-the-secure-interrupts.

What about a new directive `BL32_ROUTE_SECURE_FIQ=undefined|0|1` that would be set from `SP_MIN_WITH_SECURE_FIQ` in the **make_helpers/default.mk**. **el3_common_macros.S** would rely on `BL32_ROUTE_SECURE_FIQ` not `SP_MIN_WITH_SECURE_FIQ`.